### ISSUE TITLE
Language days,tomorrow,today should follow User Profile language

### ISSUE THREAD

**Issue by kaosmagix** [pippyn/Home-Assistant-Sensor-Afvalbeheer#349]
The language on days, tomorrow, today should be based on the language of the user settings. In this case I have my profile set to English and my spouse have it set to Dutch. Currently the labels follow the language of the config option dutch: true/false

**Comment by awolkers** [comment]
@kaosmagix funny, you've created two identical issues that I also wanted to address. 

I think the output should indeed hook into the translations already availible in HA. This could be especially usefull for foreign people who are living in the Netherlands.

**Comment by pippyn** [comment]
Yes I really want this too! I haven't found any way to do this however.
If anybody could point me in the right direction I will implement this asap.

**Comment by awolkers** [comment]
This might be a starting point.
https://developers.home-assistant.io/docs/internationalization/custom_integration

**Comment by pippyn** [comment]
I'm aware of the docs. This will not use the translations already in HA

**Comment by awolkers** [comment]
Maybe check this one as an example on how to set it up:
https://github.com/pinkywafer/Anniversaries/tree/master/custom_components/anniversaries

Not sure if you can opt in on already defined translations but you can always write you own right? I can image that for your component a bare minimum of NL, EN and FR is sufficient since the focus is on Dutch and Belgian waste collectors only.

**Comment by pippyn** [comment]
Where do you want me to look? What I can see this sensor is only in English.

(only the config flow is translated as far as I can see)

**Comment by awolkers** [comment]
I'm not that familiar with Python but can imagine the same logic for using translations within the config flow applies to the place where you set the sensor values.

**Comment by sebmatton** [comment]
Hi there! I'm really interested by this feature as well. My family is french speaking and there is no way to translate to french...

Is there no easy way to set the entities as standard date values which can be supported by HA ?

**Comment by Friedrieck** [comment]
Some info here, I think: https://developers.home-assistant.io/docs/internationalization/core#state-of-entities 
Inspiration can also be taken from here: https://github.com/olibos/HomeAssistant-RecycleApp (it does not translate the states or fractions, just get them passing the language parameter)
